msteams-platform/sbs-gs-workflow-bot.yml

```diff
@@ -196,20 +198,20 @@ items:
   durationInMinutes: 2
   content: |
     You can interact with the bot in Teams by sending a helloWorld command. After running this template, you get a response then select the **workflow bot** and **Enter**.
```

Comment:
Seems this sentence can be removed.

Reply:
I mean remove "After running this template, you get a response then select the **workflow bot** and **Enter**.", and still keep "You can interact with the bot in Teams by sending a **helloWorld** command."